methods/transient/Transient_base.ipynb

```diff
@@ -46,19 +46,25 @@
     "NISAR must be able to constrain displacements in two look directions (left-looking on both ascending and descending tracks) for the active area of 70% of target sites with an accuracy that scales with baseline distance L between any two locations within a scene. This pertains to all directly measured or inferred 12-day interferograms over the duration of the mission. Here, accuracy is calculated using L in kilometers but with the units removed, at or better than 100 m resolution. The 12-day time scale corresponds to a repeat NISAR pass on each of the ascending and descending satellite tracks, from which interferograms can be made and displacements estimated. The NISAR mission has compiled a list of 2000 global targets covering areas of known or potential transient deformation related to the processes specified in the requirement (NISAR Handbook Appendix H, 2018). These targets include all Earth's active volcanoes above sea-level, areas of rapid glacial mass changes, selected deforming reservoirs of water, oil, gas, CO2 and steam and landslide-prone areas near major population centers, as well as sites where selected disaster-related events have occurred. \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    work_dir = calval_dir/calval_location
</pre><p>This <code>str/str</code> is not Python syntax, testing on my laptop with macOS does not work. Please use <code>os.path.join()</code> for it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Sometimes this worked for me and sometimes it did not. It seems that the Python <code>Path</code> module somehow implements a way to use this syntax as long as the <code>calval_dir</code> is a Path object instead of a string. In my Coseismic notebook, I changed the variable names of the Path objects to <code>calval_dir_obj</code> to keep track of what was what, but it still sometimes does not work.   </p>

Reply:
Thank you Eric, I did not realize it's `Path` object, not `str`. As long as it works, then either way sounds good to me.